package.json

```diff
@@ -17,8 +17,8 @@
 		"cli-setup": "pnpm install && jetpack cli link",
 		"cli-unlink": "jetpack cli unlink",
 		"lint": "pnpm run lint-file .",
-		"lint-changed": "eslint-changed --ext .js,.jsx,.cjs,.mjs,.ts,.tsx,.svelte --git",
-		"lint-file": "eslint --ext .js,.jsx,.cjs,.mjs,.ts,.tsx,.svelte",
+		"lint-changed": "eslint-changed --ext .js,.jsx,.cjs,.mjs,.ts,.tsx,.svelte --eslint-options flags='[\"unstable_config_lookup_from_file\"]' --git",
+		"lint-file": "eslint --flag unstable_config_lookup_from_file",
```

Comment:
I assume extensions were removed from here because if we're linting a specific file we don't care about its extension?

Reply:
Eslint removed the `--ext` command line option, at least when using flat configs. Instead they want you to do it in the config like
https://github.com/Automattic/jetpack/blob/826194f564b6a6ed2b343611ab9e8a610beaabe0/tools/js-tools/eslintrc/base.mjs#L125-L136

This isn't just used for a single file, it's also used for full directories. `pnpm lint` even calls it with `.` as the directory.

Reply:
Ah, I hadn't got to the second commit yet. :^)

Is there a way to get `eslint-changed` to use the same config?

Reply:
There's probably a way, but it might get complex. And we'd still need to have the option for people still on non-flat configs.